### PR TITLE
fix pipeline "java-aggregate-reports" failures about spring modules

### PR DESCRIPTION
**Pull request by backwind1233**
fix the pipeline[ java-aggregate-reports](https://dev.azure.com/azure-sdk/internal/_build?definitionId=1359&_a=summary)  errors for spring modules

Ref：
https://strikecommunity.azurewebsites.net/articles/4114/credential-scanner-overview.html

### CODE REVIEW

**Review by chenrujun — APPROVED** [review]
LGTM.

File: sdk/appconfiguration/spring-cloud-azure-appconfiguration-config/src/test/java/com/microsoft/azure/spring/cloud/config/TestConstants.java [hunk 1]
```diff
@@ -45,6 +45,7 @@ private TestConstants() {
     public static final String TEST_ENDPOINT = "https://fake.test.config.io";
     public static final String TEST_KV_API = TEST_ENDPOINT + "/kv?key=fake-key*&label=fake-label";
     public static final String TEST_ID = "fake-conn-id";
+    // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="...")]
```
Comment (saragluna) [inline sdk/appconfiguration/spring-cloud-azure-appconfiguration-config/src/test/java/com/microsoft/azure/spring/cloud/config/TestConstants.java:48]: should we also provide a valid justification here?

**Review by saragluna — APPROVED** [review]
LGTM.

**Review by stliu — APPROVED** [review]

**Pull request merged**